Foxtrot/Driver/Rewriting/EmitAsyncClosure.cs

```diff
@@ -59,6 +58,121 @@ namespace Microsoft.Contracts.Foxtrot
     /// </remarks>
     internal class EmitAsyncClosure : StandardVisitor
     {
+        /// <summary>
+        /// Class that maps generic arguments of the enclosed class/method the generic arguments of the closure.
+        /// </summary>
+        /// <remarks>
+        /// The problem.
+        /// Original implementation of the Code Contract didn't support async postconditions in generics.
+        /// Here is why:
+        /// Suppose we have following function (in class <code>Foo</code>:
+        /// <code><![CDATA[
```

Comment:
@hubuk Yep, @AlexanderTaeschner is right. Otherwise we would have to encode all this `<` with `&lt;` which is annoying...